### DISCUSSION TITLE
Import Custom OHLCV Data?

### DISCUSSION THREAD

**Discussion by td2thinh** [pranjal-joshi/Screeni-py#110] [General]
I stumbled upon your project searching for a python based scanner. I got OHLCV data in Pandas Dataframe datatype for all tickers of my stock market, is there a way to import this data into the scanner instead of loading the stock data from yfinance? My stock market isnt available on yahoo finance.

**Comment by pranjal-joshi** [comment]
Hi,
You can modify https://github.com/pranjal-joshi/Screeni-py/blob/c2899ab1097802ee2b7c91078a4c197c98377f79/src/classes/Fetcher.py#L111 this method to pass your own OHLC data instead of using `yfinance`.
Just make sure that your custom dataframe column names matches the `yfinance` structure as the screener class uses these column names extensively.
Hope this helps.

**Reply by td2thinh** [reply]
Hey, thanks for the quick response, I managed to do what I needed to do, thanks.
